packages/cysync-core/src/constants/i18n/en.ts

```diff
@@ -1541,6 +1541,46 @@ Download from <a href="https://cypherock.com/get-started">https://cypherock.com/
 * No new updates to CySync v1 except security update
 `,
     },
+    editAccount: {
+      tab1: {
+        title: 'Select the Account to Edit',
+        subtitle: 'Choose the account you want to edit',
+        btn: 'Continue',
```

Comment:
No need for defining `btn`. Use the already defined variable `strings.buttons.continue`